general-configuration-guide/basic-setup.md

```diff
@@ -143,17 +143,17 @@ tinymce.init({
 
 ### A quick look at menu and menubar controls
 
-Just as there is a `toolbar` option, there is also a `menu` option. In fact, there are two menu related options: `menu` and `menubar`. Let's take a quick look at the default menu items. You will see some of these settings used in the examples throughout the Get Started guide and all of them within the plugins and customization documentation.
+Just as there is a `toolbar` option, there is also a `menu` option. In fact, there is two menu related options: `menu` and `menubar`. Let's take a quick look at the default menu items. Some of these settings are used in the examples throughout this guide and all of them within the plugins and customization documentation.
```

Comment:
I would have said "are"

Reply:
The plural form of be 'are' does not seem to agree with the singular subject `two menu`.